A brief summary of the EpiATLAS project, for display alongside the data on the IHEC Data Portal.

Loosely inspired by the Roadmap's page: [https://egg2.wustl.edu/roadmap/web_portal/](https://egg2.wustl.edu/roadmap/web_portal/), though intended to be more brief than comprehensive.

Could also be comparable to McGill EMC's methods page, though modified for EpiATLAS: [https://epigenomesportal.ca/edcc/doc/analysis_methods.html](https://epigenomesportal.ca/edcc/doc/analysis_methods.html).

* Description
* Figures
* Resources
* Articles
* BioRxiv channel: [https://connect.biorxiv.org/relate/content/219](https://connect.biorxiv.org/relate/content/219)   
* Consortium website: [https://ihec-epigenomes.org/](https://ihec-epigenomes.org/)

Data Processing Pipelines:
* ChIP-seq data was processed using the following container pipeline:
  * https://github.com/IHEC/integrative_analysis_chip/releases
* WGBS data was processed using the following container pipeline:
  * https://github.com/heathsc/gemBS/releases/tag/v3.5.0
* RNA-seq data was processed using the following container pipeline:
  * https://github.com/IHEC/grape-nf/releases

Other data sources:
* Complete processed data available through rsync/https/ftps: [https://ihec.sd4h.ca](https://ihec.sd4h.ca/)
* Metadata also in EpiRR: [https://www.ebi.ac.uk/epirr/](https://www.ebi.ac.uk/epirr/)
* Raw, unprocessed data is in EGA, DDBJ, & [https://www.encodeproject.org](https://www.encodeproject.org)
